pkg/controllers/provisioning/binpacking/packable.go

```diff
@@ -242,7 +242,7 @@ func (p *Packable) validateAWSNeurons(pods []*v1.Pod) error {
 func (p *Packable) validateAWSPodENI(pods []*v1.Pod) error {
 	for _, pod := range pods {
 		for _, container := range pod.Spec.Containers {
-			if _, ok := container.Resources.Requests[resources.AWSPodENI]; ok {
+			if _, ok := container.Resources.Limits[resources.AWSPodENI]; ok {
```

Comment:
I think you want to use Requests here. Kubernetes will default requests to the value of limits if only limits are set.

Reply:
Yup, I tested and you're right.  swapped it back.  Saw an issue when i set requests but not limit on the client side.